### PR TITLE
removes maint access from security officers outside of engie guards and skeleton crew

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
makes the 'roll around maintenance roundstart searching for a valid' technique less common or fast, it's not fun for anyone when there's 6 security officers running around maintenance roundstart and killing all the valids in like 5 minutes which ruins the game for everyone. they can of course get access added to their ID but that requires cooperation between players and at the very least will slow down the maintenance rollout
engie guard gets maintenance access because maintenance is part of engineering (technically) so it makes sense
# Changelog

:cl:  
tweak: maintenance access is now restricted to engineering security or lowpop security, otherwise they dont have it
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Wow this is a terrible idea

**Comment by TosterIMC** [comment]
Best idea ever, no more patrol maint so i can find stealth antags

**Comment by Hopekz** [comment]
This is a bad way to go about your goal.
If you are concerned with the beginning of the round why not just make their access not work for the first few minutes versus adding a chore they have to do every single shift.

I get that what I just proposed is a bad idea as well because the whole idea of security not having access to maintenance is pretty weird and probably shouldn't be merged.

**Comment by 00ze-cyclone** [comment]
WAAAAAAAAAAAH!!  SEC VALIDHUNTING ME!!
lmao

**Comment by ynot01** [comment]
officers already suffer from being pretty much useless when someone is committing crimes in an area they don't have access to

restricting them out of the maintenance areas when
- the maint area is so useful that it is essentially a requirement for any antag
- it is incredibly easy for each antag to get into maint

would be a huge nerf to security

tl;dr Wow this is a terrible idea

**Comment by Bibby0110** [comment]
This is a terrible idea

**Comment by JamieD1** [comment]
I backup Gamer.

**Comment by Simplehorror** [comment]
how are you so full of bad ideas

**Comment by Hopekz** [comment]
> how are you so full of bad ideas

It's fine as long as there is someone else to filter the Good ones from the bad ones
